core/sql/optimizer/mdam.cpp

```diff
@@ -2002,7 +2002,8 @@ void Disjuncts::computeCommonPredicates()
 	      inVidset.next(predId);
 	      inVidset.advance(predId) )
 	 {
-            if(predId.getItemExpr()->getOperatorType() == ITM_RANGE_SPEC_FUNC )
+            //TRAFODION-2988
+            /*if(predId.getItemExpr()->getOperatorType() == ITM_RANGE_SPEC_FUNC )
```

Comment:
One cosmetic issue. It is better to delete code you are replacing rather than commenting it out. The only reason to comment out code is if we are temporarily disabling it in hopes of putting it back it later.

Reply:
Thanks Dave,your suggestion is right.I will fix it and push again.